File: common/mrtjp/projectred/expansion/ExpansionRecipes.java
package mrtjp.projectred.expansion;

import mrtjp.projectred.ProjectRedExpansion;
import mrtjp.projectred.core.ItemPart.EnumPart;
import mrtjp.projectred.transmission.EnumWire;
import net.minecraft.item.Item;
import net.minecraft.item.ItemStack;
import cpw.mods.fml.common.registry.GameRegistry;

public class ExpansionRecipes {
    
    public static void initRecipes() {
        initMiscRecipes();
    }
    
    private static void initMiscRecipes() {        
        /** VAWT **/
        GameRegistry.addRecipe(new ItemStack(ProjectRedExpansion.itemVAWT, 1), 
                "sss",
                "ttt",
                "sss",
                's', EnumPart.SAIL.getItemStack(),
                't', Item.stick
        );
        GameRegistry.addRecipe(new RecipeVAWTRecoloring());
    }
}

File: src/mrtjp/projectred/expansion/ExpansionRecipes.scala
package mrtjp.projectred.expansion

import java.lang.{Character => JC}

import cpw.mods.fml.common.registry.GameRegistry
import mrtjp.projectred.ProjectRedExpansion._
import net.minecraft.init.{Blocks, Items}
import net.minecraft.item.ItemStack
import net.minecraftforge.oredict.ShapedOreRecipe

object ExpansionRecipes
{
    def initRecipes()
    {
        //FurnaceRecipeLib.init()
        initMachineRecipes()
        initMiscRecipes()
    }

    private def initMiscRecipes()
    {
    }

    private def initMachineRecipes()
    {
        //Block Breaker
        GameRegistry.addRecipe(new ShapedOreRecipe(new ItemStack(machine2, 1, 0),
            "sas", "sps", "srs",
            's':JC, Blocks.cobblestone,
            'a':JC, Items.iron_pickaxe,
            'p':JC, Blocks.piston,
            'r':JC, Items.redstone
        ))

        //Block Breaker
        GameRegistry.addRecipe(new ShapedOreRecipe(new ItemStack(machine2, 1, 1),
            "www", "sps", "srs",
            'w':JC, "plankWood",
            's':JC, Blocks.cobblestone,
            'p':JC, Blocks.piston,
            'r':JC, Items.redstone
        ))
    }
}